### PR TITLE
Do not ask username

### PR DESCRIPTION
**Pull request by ultroned**
We can get all user info from GH API.

### CODE REVIEW

**Comment by ultroned** [comment]
@m-k please review

File: app/controllers/authentication_controller.rb [hunk 1]
```diff
@@ -39,4 +32,8 @@ def login_error(message)
     flash[:notice] = message
     redirect_to_login
   end
+
+  def load_user
+    GithubActions::LoadUserService.new(params[:user][:token]).perform
```
Comment (m-k) [inline app/controllers/authentication_controller.rb]: this method used three times, pls memoize it or use `:before_action` for loading user into instance variable.

**Review by m-k — COMMENTED** [review]
@loogle18 please fix `load_user`.

**Comment by ultroned** [comment]
@m-k done. Please re-check.

**Pull request merged**